### ISSUE TITLE
add manual resegmentation 

### ISSUE THREAD

**Issue by diemoschwarz** [Ircam-RnD/catart-mubu#48]
With mubu v1.10.0 and https://git.forum.ircam.fr/haddad/mubu/-/issues/136, pipo.chop accepts manually edited segmentation times.  For resegmentation, we then need to keep the raw instantaneous descr. data around.

Todo
- [ ] decide on now necessary new track name
   - `descr` for new raw instantaneous descriptors, `segments` for segmentation output with mean descr. values? This would break compatibility for all existing patches.
   - `rawdesr` or `instdescr` for raw instantaneous descriptors, `descr` for segments and mean values as before?
- [ ] integrate into main catart-mubu-simple patch in markers view
   - [ ] with auto-recalc toggle
   - [ ] raw descr. in multibpf, off by default (spectrogram view would need to be scaled)
- [ ] add a tutorial

**Comment by diemoschwarz** [comment]
Hi Aaron @einbond and Chris @christophertrapani, what do you think would be clear new track names (I'd trend towards the second variant, not breaking the existing patches)?

**Comment by einbond** [comment]
I definitely agree not to break the existing patches! ;) So descr remains as before, and I think rawdescr or just raw would be good for the new name.

A further idea toward backward compatibility: would there be a way to keep the old workflow with a single mubu.process when manual segmentation is not needed, and treat the new version with two mubu.process objects as a special case? Basically not to create the extra track and structure if it is not going to be used.

The camu.process-* patches are modular anyway, so there could be alternative versions that implement the double mubu.process architecture, and the user can switch dynamically if needed, the same way one can switch between different segmentation algorithms or descriptor calculations.

**Comment by diemoschwarz** [comment]
Thanks for the input, Aaron! You're right that the `camu.process.*` patches could stay as they are, and, in the main patches with resegmentation, a new analysis subpatch pull in bpatchers with a different prefix.  What about `camu.process2.*`?
However, that would then make 3 sets of segmentation methods to be maintained.

**Comment by christophertrapani** [comment]
Hi guys — Yes, let's use something like "rawdescr" and not break the
patches. Totally in agreement. Looking forward to delving back into
catart.mubu soon — C

**Comment by einbond** [comment]
I agree with the approach pulling in multiple bpatchers and naming the second set `camu.process2.*` There are currently 6 modules (3 segmentation methods x 2 descriptor calculations, descr and ircamdescriptors) but we could decide if copies of all of them are needed—maybe just start with the simpler descr modules? If we have a model of what the two mubu.process objects would look like in the catart-mubu-simple patch I can adapt it in the bpatchers.

**Comment by diemoschwarz** [comment]
On adding resegmentation to tut 2b (it works!), it occurred to me that we can now split descr. analysis from segmentation.
I.e. have a choice of _descr, mfcc, ircamdescriptors_ bpatchers and then the segmentation choice.
The former just have to transmit column index and extent for loudness to the segmenters.

So we'd only have 6 new modules `camu.process2-*` with `*` being `descr, mfcc, ircamdescriptors, chop, gate, onseg`, where otherwise it would have been 9!  

I'd say that we shouldn't keep the `camu.process2.descr-onseg`, etc., variants around, then.

this warrants an issue of its own: #49
